{
  "bytearray": [
    [],
    {
      "__error__": {
        "type": "TypeError",
        "message": "string argument without an encoding"
      }
    },
    [
      195,
      164,
      115,
      100,
      102
    ],
    [
      0,
      0,
      0,
      0,
      0,
      0,
      0,
      0,
      0,
      0
    ],
    [
      1,
      5,
      10
    ],
    {
      "__error__": {
        "type": "TypeError",
        "message": "'str' object cannot be interpreted as an integer"
      }
    },
    {
      "__error__": {
        "type": "ValueError",
        "message": "byte must be in range(0, 256)"
      }
    }
  ],
  "bytes": [
    [],
    {
      "__error__": {
        "type": "TypeError",
        "message": "string argument without an encoding"
      }
    },
    [
      195,
      164,
      115,
      100,
      102
    ],
    [
      0,
      0,
      0,
      0,
      0,
      0,
      0,
      0,
      0,
      0
    ],
    [
      1,
      5,
      10
    ],
    {
      "__error__": {
        "type": "TypeError",
        "message": "'str' object cannot be interpreted as an integer"
      }
    },
    {
      "__error__": {
        "type": "ValueError",
        "message": "bytes must be in range(0, 256)"
      }
    }
  ],
  "dict": [
    {},
    {},
    {
      "__error__": {
        "type": "TypeError",
        "message": "dict expected at most 1 argument, got 2"
      }
    },
    {
      "a": 1,
      "b": 2
    },
    {
      "__error__": {
        "type": "TypeError",
        "message": "cannot convert dictionary update sequence element #0 to a sequence"
      }
    },
    {},
    {
      "a": 1,
      "b": 2
    },
    {},
    {
      "__error__": {
        "type": "ValueError",
        "message": "dictionary update sequence element #0 has length 1; 2 is required"
      }
    },
    {
      "__error__": {
        "type": "TypeError",
        "message": "'int' object is not iterable"
      }
    },
    {
      "__error__": {
        "type": "TypeError",
        "message": "dict expected at most 1 argument, got 2"
      }
    },
    {
      "__error__": {
        "type": "TypeError",
        "message": "'NoneType' object is not iterable"
      }
    }
  ],
  "range": [
    [
      0,
      1,
      2,
      3,
      4
    ],
    [],
    [
      1,
      2,
      3,
      4,
      5,
      6,
      7,
      8,
      9
    ],
    [
      1,
      3,
      5,
      7,
      9
    ],
    [],
    [],
    [],
    [
      -1,
      -4,
      -7
    ],
    [
      -10,
      -9,
      -8,
      -7,
      -6,
      -5,
      -4,
      -3,
      -2
    ],
    [
      -10,
      -8,
      -6,
      -4,
      -2
    ],
    [],
    {
      "__error__": {
        "type": "ValueError",
        "message": "range() arg 3 must not be zero"
      }
    }
  ]
}